tests/jenkins/pages/run.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash -ex
-
-# shellcheck source=/dev/null
-source ./venv_modules/bin/activate
-firefox-ui-functional tests/test_tp.py --binary firefox/FirefoxNightly.app/Contents/MacOS/firefox-bin -v
```

Comment:
I'm guessing this is this now not used due to the switch to the new Jenkins environment?

Reply:
We're actually not sure what this is from, but it's never been used in relation to these tests.